lib/index.js

```diff
@@ -539,7 +539,7 @@ function _getPublicKeyForPermissionCheck(publicKey, options, callback) {
   if(typeof publicKey === 'string') {
     publicKey = {id: publicKey || ''};
   }
-  api.getPublicKey(null, publicKey, function(err, publicKey) {
+  api.getPublicKey(publicKey, null, function(err, publicKey) {
```

Comment:
`getPublicKey` defaults actor to null if you just pass in two args.  How is that different?